pkg/controller/scansettingbinding/scansettingbinding_controller.go

```diff
@@ -429,7 +429,8 @@ func (r *ReconcileScanSettingBinding) verifyRoleInScanSettingBinding(
 
 		profileType, err := getScanType(tp.GetAnnotations())
 		if err != nil {
-			return fmt.Errorf("error getting profile type %s: %w", prfRef.Name, err)
+			logger.Info("error getting scan type, assuming Platform", "error", err)
```

Comment:
It might still be valuable to log the profile name.

Reply:
sure, done